config/amsterdam.config.js

```diff
@@ -54,7 +56,15 @@ export default {
             "longitude": 4.8952
         },
         "zoom": 12,
-        "attribution": true,
-        "extent":  [ 4.7283, 52.2771, 5.0799, 52.4324 ]
+        "attribution": false,
+        "extent":  [ 4.7283, 52.2771, 5.0799, 52.4324 ],
+        "zoomposition": "bottomright"
+    },
+    "classnames": {
```

Comment:
You're still using this?

Reply:
yes, these are additions especially for the Amsterdam viewer.